iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -153,6 +156,24 @@ public DecomposedPredicate decomposePredicate(JobConf jobConf, Deserializer dese
     return predicate;
   }
 
+  @Override
+  public Map<String, String> getBasicStatistics(TableDesc tableDesc) {
```

Comment:
this method seem to be using a `TableDesc` to be able to identify the underlying Iceberg table - wouldn't it be possible to do the same from a  `Table` object?

Reply:
We need the TableDesc, since the properties which are required to load the iceberg table are stored there.